Solutions/NetClean ProActive/Analytic Rules/NetClean_Sentinel_analytic_rule.yaml

```diff
@@ -0,0 +1,42 @@
+id: 77548170-5c60-42e5-bdac-b0360d0779bb
+name: NetClean ProActive Incidents
+description: |
+  'NetClean Incident'
+severity: High
+requiredDataConnectors: []
```

Comment:
It is mandatory to have atleast one required data connector with associated datatype in the analytics.